### DISCUSSION TITLE
Setting MotionCanvas skiaElement with a template

### DISCUSSION THREAD

**Discussion by Kay-STL** [Live-Charts/LiveCharts2#1665] [Q&A] [answered]
Hi there,
Just looking for a bit of guidance here on the same topic as #1645. 

Is it possible to change the underlying `skiaElement` in a CartesianChart -> MotionCanvas using control templates? I've been experimenting with using `SKGLView` and `SKMetalView` instead of `SKCanvasView` for hardware acceleration, and while I can modify the `SkiaSharpView` to switch, it'd be a lot better if I could use a template.

Unfortunately, everything I try seems to result in a blank chart. 
Any advice would be greatly appreciated :)

**Comment by beto-rodriguez** [comment, marked as answer]
Hi, the MotionCanvas class has some unnecessary code... maybe that is why you are facing issues, that will be for sure updated to remove all the garbage there...

I was able to use the `SKGLView` on this branch:

https://github.com/beto-rodriguez/LiveCharts2/tree/hardware-accelerated

Please feel free to try it and let me know your results 😃

**Reply by Kay-STL** [reply]
Ah, thanks! Looks great, I'll have a play around with that branch when I can.
